packages/rollup-full-node/src/app/test-handler.ts

```diff
@@ -0,0 +1,102 @@
+/* External Imports */
+import { add0x, getLogger, remove0x } from '@eth-optimism/core-utils'
+import { OPCODE_WHITELIST_MASK } from '@eth-optimism/ovm'
+
+import { createMockProvider, getWallets } from 'ethereum-waffle'
+import { Contract, Wallet } from 'ethers'
+import { Web3Provider } from 'ethers/providers'
+
+/* Internal Imports */
+import { DEFAULT_ETHNODE_GAS_LIMIT } from './index'
+import { DefaultWeb3Handler } from './handler'
+import { UnsupportedMethodError, Web3RpcMethods } from '../types'
+
+const log = getLogger('test-web3-handler')
+
+/**
+ * Test Handler that provides extra functionality for testing.
+ */
+export class TestWeb3Handler extends DefaultWeb3Handler {
```

Comment:
Fan of the approach!

Btw one thing that I've noticed when showing folks our codebase is `handler.ts` is a bit unspecific. Could consider changing both `handler.ts` and this file to `web3-handler.ts` // `web3-test-handler.ts` ? Or maybe even more specific `web3-rpc-handler.ts` and `web3-rpc-test-handler.ts`?